internal/ignition/discovery_test.go

```diff
@@ -728,6 +730,93 @@ var _ = Describe("IgnitionBuilder", func() {
 		})
 	})
 
+	const (
```

Comment:
Given that you have a bunch of setup/helpers that are specific to your tests, maybe here is a good place to create a separate context for them.

Reply:
It was suupposed to part of `mirror registries config` context. I'll fix taht